.prettierrc

```diff
@@ -0,0 +1,6 @@
+printWidth: 100
```

Comment:
do we want to include prettier in this PR? also should we enforce it in travis that everything needs to be prettiefied?

Reply:
I think that's worth a separate discussion and PR if we decide to move that way.